src/System.IO.Compression/tests/ZipArchive/zip_CreateTests.cs

```diff
@@ -149,6 +149,88 @@ public static void CreateUncompressedArchive()
             }
         }
 
+        [Theory]
+        [InlineData("minimalTwoFiles")]
```

Comment:
```suggestion
        [InlineData("minimalTwoFiles")]
```
This line ended up being unnecessary after I realized adding new files would be overkill.